xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -443,7 +445,7 @@ bool CPVRClient::GetAddonProperties(void)
         types_array[size].iAttributes = PVR_TIMER_TYPE_IS_MANUAL               |
                                         PVR_TIMER_TYPE_SUPPORTS_ENABLE_DISABLE |
                                         PVR_TIMER_TYPE_SUPPORTS_CHANNELS       |
-                                        PVR_TIMER_TYPE_SUPPORTS_START_END_TIME |
+                                        PVR_TIMER_TYPE_SUPPORTS_START_TIME     |
```

Comment:
Good catch - I was intending to add them into the split anytime commit but forgot them. Will do.